views.go

```diff
@@ -21,6 +24,32 @@ import (
 
 const maxLogBytes = 4 * 1024 * 1024 // 4 MB
 
+const (
+	corsOriginsEnvVariable = "LK_CORS_ORIGINS"
+)
+
+var corsOrigins []string
+
+func init() {
```

Comment:
I think this would still be compiling the regexes for each request since `utility.StringMatchesAnyRegex` calls `regexp.MatchString` which Compiles the string regex.